App.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import List from './List'
+import Dashboard from './Dashboard'
+
+class App extends Component {
+  constructor() {
+    super;
+
+    this.state ={ view: 'main', };
+  }
+
+  componentWillMount() {
+    fetch('/api/users')
+    .then(res => res.json())
+    .then(res => {
+      let data = res.data;
+      this.setState({users: data })
+      this.setState({view: 'list'})
+    })
+  }
+  
+  render() {
+    let {view,list} = this.state;
```

Comment:
please fix the spacing here, use const since you won't be mutating these variables
```suggestion
    const { view, list } = this.state;
```